components/carbon-transports/websocket/org.wso2.carbon.websocket.transport/src/main/java/org/wso2/carbon/websocket/transport/SubprotocolBuilderUtil.java

```diff
@@ -45,8 +45,12 @@ public static String syanapeSubprotocolToContentType(String subprotocol) {
     }
 
     public static String contentTypeToSyanapeSubprotocol(String contentType) {
-        return SYNAPSE_SUBPROTOCOL_PREFIX + SYNAPSE_CONTENT_TYPE + "'" + contentType +
-                "'" + SYNAPSE_SUBPROTOCOL_SUFFIX;
+        if (contentType.equals("graphql-ws")) {
```

Comment:
Move the string "graphql-ws" to a String constant similar to SYNAPSE_CONTENT_TYPE